### PR TITLE
Add ProGuard support

### LINKED ISSUE #172: Proguard can't find referenced class FragmentCompat
Hi!

In this file: https://github.com/hotchemi/PermissionsDispatcher/blob/master/library/src/main/android/permissions/dispatcher/v13/V13Access.java

you are referencing android.support.v13.app.FragmentCompat
When using your library with ProGuard there are 3 warnings preventing a successful build:

```
permissions.dispatcher.v13.V13Access: 
can't find referenced class android.support.v13.app.FragmentCompat
```

I already tried to add proguard rules like

```
-keep public class android.support.v13.app.FragmentCompat { *; }
-keep public class permissions.dispatcher.v13.** { *; }
```

but to no avail. Do you have any hint, how to get the library to work with proguard?

### PR DESCRIPTION
**Pull request by shiraji**
Close #172 

I made draft for README. Feel free to comment on that!

### CODE REVIEW

**Comment by hotchemi** [comment]
Cool, let me check tomorrow! Good night!

**Comment by shiraji** [comment]
Have a wonderful night!

File: README.md [hunk 1]
```diff
@@ -111,6 +111,10 @@ Simply add a dependency on the `support-v13` library alongside PermissionsDispat
 
 - [Migrating to 2.x](https://github.com/hotchemi/PermissionsDispatcher/wiki/Migrating-to-2.x)
 
+## ProGuard
+
+PermissionsDispatcher takes care of ProGuard. No extra setting is required.
```
Comment (hotchemi) [inline README.md]: Good draft! Can you possibly fix like following? 🙇 
`PermissionsDispatcher bundles ProGuard setting in aar. No extra setting is required.`

**Comment by hotchemi** [comment]
Almost LGTM!

File: README.md (hunk 1 above)
Comment (aurae) [inline README.md]: Here's mine:
`PermissionsDispatcher bundles ProGuard rules in its aar. No extra settings are required.`

Comment (shiraji) [inline README.md]: Thanks English teachers!

Comment (shiraji) [inline README.md]: Fix and squash it

**Comment by hotchemi** [comment]
@shiraji Thx man!

**Pull request merged**